### PR TITLE
changes for using fake docking and power usage

### PR DESCRIPTION
**Pull request by souravran**

### CODE REVIEW

File: cob_bringup/tools/docker_control.launch [hunk 1]
```diff
@@ -32,4 +33,8 @@
 	<node pkg="cob_docker_control" type="docker_control.py" respawn="false" name="docker_control" output="screen" unless="$(arg multi_station)">
 	</node>
 
+	<!-- virtual charging and power usage for simulation -->
+	<node if="$(arg sim)" pkg="cob_docker_control" type="fake_docking.py" respawn="false" name="fake_docker_control" output="screen" >
+	</node>
```
Comment (fmessmer) [inline cob_bringup/tools/docker_control.launch]: can we use single line notation?
`<node if="$(arg sim)" pkg="cob_docker_control" type="fake_docking.py" name="fake_docker_control" respawn="false" output="screen" />`

File: cob_bringup/tools/docker_control.launch [hunk 2]
```diff
@@ -32,4 +33,8 @@
 	<node pkg="cob_docker_control" type="docker_control.py" respawn="false" name="docker_control" output="screen" unless="$(arg multi_station)">
 	</node>
```
Comment (fmessmer) [inline cob_bringup/tools/docker_control.launch]: This node does not exist anymore...what happened to it?

File: cob_bringup/tools/docker_control.launch (hunk 1 above)
Comment (fmessmer) [reply cob_bringup/tools/docker_control.launch]: Not sure, but does any of the other nodes than need a `unless="$(arg sim"` tag?

**Review by fmessmer — CHANGES_REQUESTED** [review]

File: cob_bringup/tools/docker_control.launch (hunk 2 above)
Comment (souravran) [reply cob_bringup/tools/docker_control.launch]: I had the same question but my best guess is that the node may be obsolete or has been updated to ``multi_station_docker_control.py`` and was not removed from here.

File: cob_bringup/tools/docker_control.launch (hunk 1 above)
Comment (souravran) [reply cob_bringup/tools/docker_control.launch]: Even I am not sure about the ``unless="$(arg sim)"`` tag, simulation works fine though.

File: cob_bringup/tools/docker_control.launch (hunk 2 above)
Comment (fmessmer) [reply cob_bringup/tools/docker_control.launch]: Do you see any error message about "Node not there"?

Please remove the node from the launch file

Comment (souravran) [reply cob_bringup/tools/docker_control.launch]: No I dont get any error. Alright.

Comment (fmessmer) [reply cob_bringup/tools/docker_control.launch]: There still is a `docker_control.py` in https://github.com/ipa320/cob_substitute/tree/indigo_dev/cob_docker_control/scripts, maybe it uses this one...
Do you have an overlay of `cob_substitute`? Or do you use a released version?
Can you put one in your workspace and remove the `docker_control.py`? Then you should get an error....
Please provide a PR for cob_substitute where the script is removed...

In the same way, we might need to add a `fake_docking.py` to `cob_substitute`!

Comment (souravran) [reply cob_bringup/tools/docker_control.launch]: No I dont have cob_substitute, will open an PR after testing with cob_substitute overlay.

**Comment by fmessmer** [comment]
requires https://github.com/ipa320/cob_driver/pull/321 for travis to be happy

@ipa-fmw 
Do we need to `unless=$(arg sim)` the `multi_station_docker_control.py`?
(What's the purpose of this file anyway?)

@souravran 
PR to `cob_substitue` still missing

**Comment by floweisshardt** [comment]
we typically use multi_docking, which can dock e.g. to the charging station and the travel box. This works too for simulation (together with fake_docking providing correct power_state), so no need for `arg sim` in multi_docking.
But we still need `arg sim` for fake_docking

**Review by fmessmer — APPROVED** [review]
cob_substitute has been updated: https://github.com/ipa320/cob_substitute/pull/43

**Pull request merged**